ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_isis.py

```diff
@@ -90,3 +97,17 @@ def _is_type(self) -> str:
         if is_type not in ["level-1", "level-2", "level-1-2"]:
             is_type = "level-2"
         return is_type
+
+    @staticmethod
+    def ipv4_to_isis_system_id(ipv4_address):
+        """
+        Converts an IPv4 address into an IS-IS system-id.
+
+        Examples:
+        192.168.0.1 -> 1921.6800.0001
+        10.0.0.3 -> 0100.0000.0003
+        """
+        octets = ipv4_address.split(".")
```

Comment:
Are we sure that `ipv4_address` will always be a string? Otherwise this could throw an `AttributeError`

Reply:
fixed.